src/WidenImage.js

```diff
@@ -0,0 +1,86 @@
+import './WidenImage.css'
+import utilities from './Utilities'
+
+// If enabled, widened images will have thin red dashed border
+const enableDebugBorders = false
```

Comment:
If the intent is to allow runtime debugger changes to this value, do we need this to be `let` (with an `// eslint-disable-line prefer-const`) instead of `const`? In Chromium, when I type `const foo = 0; foo = 1` it throws a `TypeError`.